lib/x11test.pm

```diff
@@ -874,6 +874,18 @@ sub configure_static_ip_nm {
     wait_screen_change { send_key 'alt-f4' };
 }
 
+# Open the firewall port of xdmcp service
+sub configure_xdmcp_firewall {
+    # Open the firewall port of xdmcp service
+    if (is_sle '15+') {
```

Comment:
JFYI, we have firewall property defined, so in case it's specific to firewalld, we can simply write:
```
$self->firewall eq 'firewalld'
```

Reply:
Oh, yes. Please change that

Reply:
fixed by: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4825